### PR TITLE
Spelling: All language reworked

### PR DESCRIPTION
**Pull request by comradekingu**
Hopefully this saves quite a bit of time in getting https://hosted.weblate.org/projects/dont-kill-my-app in shape.

### CODE REVIEW

File: _vendors/xiaomi.md [hunk 1]
```diff
@@ -76,9 +76,14 @@ Enable Autostart for desired apps.
...
-When you open recent apps tray, drag your app downwards – it will be locked. So even if you clear recent apps it will not clear from the background. Drag downwards again to clear your app from the background.
+When you open the recent apps tray, drag your app downwards (it will be locked). So, even if you clear recent apps, it will not clear them from the background. Drag downwards again to clear your app from the background.
 
 '
+"
+
+developer_solution: "
 
-developer_solution: "At the moment we don't know of any solution on dev end."
+So far, no workarounds on the dev side are known.
```
Comment (comradekingu) [inline _vendors/xiaomi.md:86]:
```suggestion
No known solution on the dev end.
```

File: _vendors/meizu.md [hunk 2]
```diff
@@ -18,6 +18,6 @@ Adjust your settings:
 
 * Enable **Device Settings > Apps > your app > Battery > Power-intensive prompt** and **Keep running after screen off**
 "
-developer_solution: "No known solution so far on developer end"
+developer_solution: "No known solution so far on the dev end"
```
Comment (comradekingu) [inline _vendors/meizu.md:21]:
```suggestion
developer_solution: "No known solution on the dev end"
```

File: _vendors/xiaomi.md [hunk 3]
```diff
@@ -76,9 +76,14 @@ Enable Autostart for desired apps.
 
 ### App pinning
 
-When you open recent apps tray, drag your app downwards – it will be locked. So even if you clear recent apps it will not clear from the background. Drag downwards again to clear your app from the background.
+When you open the recent apps tray, drag your app downwards (it will be locked). So, even if you clear recent apps, it will not clear them from the background. Drag downwards again to clear your app from the background.
 
 '
+"
+
+developer_solution: "
```
Comment (comradekingu) [inline _vendors/xiaomi.md:84]:
```suggestion
developer_solution: "No known solution on the dev end."
```

File: _vendors/xiaomi.md [hunk 4]
```diff
@@ -76,9 +76,14 @@ Enable Autostart for desired apps.
...
-When you open recent apps tray, drag your app downwards – it will be locked. So even if you clear recent apps it will not clear from the background. Drag downwards again to clear your app from the background.
+When you open the recent apps tray, drag your app downwards (it will be locked). So, even if you clear recent apps, it will not clear them from the background. Drag downwards again to clear your app from the background.
 
 '
+"
+
+developer_solution: "No known solution on the dev end."
 
-developer_solution: "At the moment we don't know of any solution on dev end."
+No known solution on the dev end.
```
Comment (comradekingu) [inline _vendors/xiaomi.md:86]:
```suggestion
```

File: _vendors/xiaomi.md [hunk 5]
```diff
@@ -76,9 +76,13 @@ Enable Autostart for desired apps.
...
+When you open the recent apps tray, drag your app downwards (it will be locked). So, even if you clear recent apps, it will not clear them from the background. Drag downwards again to clear your app from the background.
 
 '
+"
+
+developer_solution: "No known solution on the dev end."
 
-developer_solution: "At the moment we don't know of any solution on dev end."
+
+"
```
Comment (comradekingu) [inline _vendors/xiaomi.md:87]:
```suggestion
```

File: README.md [hunk 6]
```diff
@@ -2,22 +2,6 @@
 
 ### [dontkillmyapp.com](https://dontkillmyapp.com)
 
-## What is this about?
-
```
Comment (comradekingu) [inline README.md:6]: It is a bit of a faff to maintain duplicated info, and it is not good for SEO.

**Comment by Artaud** [comment]
Big thanks! I've done a (hopefully) thorough review, accepted as much as I could and fixed some issues mainly with unescaped quotes and with content that changed since you've made this PR (sorry it took so long). Thanks again!

**Pull request closed without merging**